robberies/fleeca-bank-robbery.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Fleeca Bank Robbery
+---
+
+**Fleeca Bank Robbery**  
+
+Adaugat un nou tip de jaf - jaf armat la banca. Jucatorii pot jefui micile banci din orasul Los Santos.  
+Acest tip de rob este unul mai complex, drept urmare jucatorii au nevoie de o arma cu munitie, C4 Bomb, Drill & Pliers.  
+
+Noile iteme pot fi dropate la job-urile de mai jos:  
+
+C4 Bomb se craftează pe insula Cayo Perico din:  
+ - 2x Explosive  
+ - 3x Wires  
+ - 1x Electronic Keyboard  
+
```

Comment:
**De ce am nevoie pentru a jefui banca?**

Pentru a jefui banca vei avea nevoie de o arma cu munitie, C4 Bomb, Drill & Pliers. „C4 Bomb” se poate crafta pe insula „Cayo Perico”, iar pentru a crafta vei avea nevoie de "2x Explosive"; "3x Wires" & "1x Electronic Keyboard.". Aceste iteme se pot dropa la urmatoarele joburi: Trucker -> Explosive: sansa de drop de 20% la fiecare $300 castigati in urma efectuarii unei curse ilegale; Electrician -> Wires: sansa de drop de 20% la fiecare $300 castigati la job & Electronic Keyboard: sansa de drop de 20% la fiecare $900 castigati la job.